### ISSUE TITLE
Support for ENS Names when adding users

### ISSUE THREAD

**Issue by reeserj** [coordinape/coordinape#1025]
**Problem**
Coordinape Doesn't support ENS names yet, this requires users to enter their 0x address, and makes spoofing users easier.

**Solution**
Add support for ENS names when adding users to Coordinape.

This would also need to be updated here: 
![image](https://user-images.githubusercontent.com/82838264/173156592-332eac85-138b-408a-8a08-5352ee90974d.png)
and here 
![image](https://user-images.githubusercontent.com/82838264/173156617-6cc857b2-c014-412a-94fa-d39323593182.png)
and here
![image](https://user-images.githubusercontent.com/82838264/173156643-bc98f21d-4afa-4af2-8352-6af3a88f9fc4.png)

**Specification**
Add support for ENS names within Coordinape and update addresses where it's displayed.
- Admin page
- Mobile UI under the hamburger menu
- Desktop UI under the PFP menu

**Stakeholders**
@spydermonkee for timeline of dev work @AlexDistill for design

**Comment by crabsinger** [comment]
Some remaining questions:
- can admins change the ENS username, or is it fixed and read from ENS on page load? When?
- Do we show the ENS name or the address, and when?
-
I think best approach is to use the ENS name as the username, and resolve the ENS to the underlying address.

**Comment by crabsinger** [comment]
@AlexDistill want to review this and share any thoughts?

**Comment by reeserj** [comment]
@sshmm if you get through with your other issues this might be a good one to look at too!

**Comment by CryptoGraffe** [comment]
I think this is supported now with the new add members screens, but it doesn't address the name vs .ens discussion that is in this issue. We should decide what we want to do. I'm not sure we should lock ENS->name.

**Comment by crabsinger** [comment]
I think we should resolve the ENS name to an address, and store the name as the ens name, the address as the resolved address in our DB.

**Comment by sshmm** [comment]
it is possible to use ens as address in nominations and adding members

**Comment by crabsinger** [comment]
> it is possible to use ens as address in nominations and adding members but they will appear as resolved address once added

Yes, I think this is what is currently implemented for add users that @CryptoGraffe did...
